test/e2e/README.md

```diff
@@ -0,0 +1,15 @@
+# End-to-end tests
+## Executing tests
+### Using ginkgo locally
+From the velero directory:
+hack/tools/bin/ginkgo test/e2e
+
+Command line flags can be set after --
+velerocli - the velero CLI to use
+kibishiins - the namespace to install kibishii in
+cloudplatform - the cloud platform the tests will be run against (aws, vsphere, azure)
+
+```
+hack/tools/bin/ginkgo test/e2e/ -- -velerocli=/usr/bin/velero
```

Comment:
I don't see the changes I made to add make targets 
```
## Running tests locally

1. From Velero repository root

    ```
    make test-e2e
    ```

1. From `test/e2e/` directory

    ```
    make run
    ```

## Running tests based on cloud platforms

1. Running Velero E2E tests on KinD

    ```
    GINKGO_FOCUS=kind make test-e2e
    ```

1. Running Velero E2E tests on AWS

    ```
    GINKGO_FOCUS=aws make test-e2e
    ```

1. Running Velero E2E tests on Azure

    ```
    GINKGO_FOCUS=azure make test-e2e
    ```

```

Reply:
I also added a make target
```
.PHONY: test-e2e
test-e2e:
	$(MAKE) -C test/e2e run
```
to the `Makefile` at the repor root.

Reply:
Hmmm...I just squashed everything down and I didn't notice any changes to the top level makefile.  Please push those again and make sure to base off the latest.

Reply:
I just pushed the changes again.